src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/model/ccd/mappers/SupplementaryEvidenceMapper.java

```diff
@@ -15,4 +19,29 @@ public SupplementaryEvidenceMapper() {
     public SupplementaryEvidence mapEnvelope(Envelope envelope) {
         return new SupplementaryEvidence(mapDocuments(envelope.documents));
     }
+
+    @Override
+    ScannedDocument mapDocument(Document document) {
+        if (StringUtils.isNotEmpty(document.subtype)) {
```

Comment:
```suggestion
      String documentSubType=StringUtils.isNotEmpty(document.subtype) ? document.subtype : null;

return new ScannedDocument(
                document.fileName,
                document.controlNumber,
                documentSubType,
                null,
                getLocalDateTime(document.scannedAt),
                new CcdDocument(document.url),
                null
            );
```

Reply:
👍 will make this chnage

Reply:
I've changed the code to keep the `type` value when subtype is null